app/assets/stylesheets/content.scss

```diff
@@ -33,9 +72,29 @@
 }
 
 #upvoted {
-  color: #BDD860;
+  color: $green;
 }
 
 #downvoted {
-  color: #D85B36;
+  color: $red;
+}
+
+/************** UPVOTES *****************************/
+
+.sidebar {
+  border-left: 1px solid $neutral;
+}
+
+/************** MEDIA QUERIES *****************************/
+
+// Small screens
+// @media only screen { } /* Define mobile styles */
+
+@media only screen and (min-width: 64.063em) {
+
+#content {
```

Comment:
Avoid using id selectors<br>Line should be indented 2 spaces, but was indented 0 spaces